batch/batch/server/deserialize.py

```diff
@@ -0,0 +1,91 @@
+from datetime import date, datetime
+import re
+from kubernetes.client import models
+
+
+def deserialize(api_client, data, klass):  # pylint: disable=R0911
+    if data is None:
+        return None
+
+    if isinstance(klass, str):
```

Comment:
The Kubernetes source uses `type(klass, str)` . `isinstance` checking base class probably won't have any unwanted side effects, but thought I'd mention it.

Reply:
These changes are because of flake8 and pylint.